enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/embed.v1.unit.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { MetabaseEmbed } from "./embed.v1";
+
+describe("embed.v1.js script tag for sdk iframe embedding", () => {
+  const defaultSettings = {
+    apiKey: "test-api-key",
+    instanceUrl: "http://localhost:3000",
+    target: "#embed-container",
+  };
+
+  it("throws when target element is not found", () => {
+    expect(() => {
+      new MetabaseEmbed({ ...defaultSettings, target: "#not-existent-target" });
+    }).toThrow('cannot find embed container "#not-existent-target"');
+  });
+
+  it("throws when target element is undefined", () => {
+    expect(() => {
+      // @ts-expect-error -- we are testing for incorrect configuration
+      new MetabaseEmbed({ ...defaultSettings, target: undefined });
+    }).toThrow('cannot find embed container "undefined"');
+  });
+
+  it("throws when api key is not provided", () => {
+    expect(() => {
+      // @ts-expect-error -- we are testing for incorrect configuration
+      new MetabaseEmbed({ ...defaultSettings, apiKey: undefined });
+    }).toThrow("api key and instance url must be provided");
```

Comment:
user will likely see this error right? In that case.

```suggestion
    }).toThrow("API key and instance URL must be provided");
```

Reply:
Fixed in https://github.com/metabase/metabase/pull/57269/commits/a0e50975dd233e086745a21a8a0fee9613714cfa, thanks!